tests/nn/data_parallel/test_fsdp_input.py

```diff
@@ -28,53 +40,45 @@
     "fsdp_config", [{}, {"mixed_precision": True}],
 )
 @pytest.mark.parametrize("input_cls", [dict, list])
-def test_it(fsdp_config, input_cls):
+def test_input_type(temp_files, fsdp_config, input_cls):
     """Test FSDP with input being a list or a dict, only single GPU."""
-    if torch_version() < (1, 6, 0):
-        pytest.skip("older pytorch doesn't support reduce_scatter")
-
-    # Random port in case the next test run quickly, same port would cause conflict.
-    os.environ["MASTER_ADDR"] = "localhost"
-    os.environ["MASTER_PORT"] = str(random.randint(2000, 3000))
-    torch.distributed.init_process_group(backend="nccl", rank=0, world_size=1)
-
-    try:
-        assert isinstance(fsdp_config, dict), str(fsdp_config)
-
-        class Model(Module):
-            def __init__(self):
-                super().__init__()
-                self.layer = Linear(4, 4)
-
-            def forward(self, input):
-                if isinstance(input, list):
-                    input = input[0]
-                else:
-                    assert isinstance(input, dict), input
-                    input = input["in"]
-                return self.layer(input)
-
-        model = FSDP(Model(), **fsdp_config).cuda()
-        optim = SGD(model.parameters(), lr=0.1)
-
-        for _ in range(5):
-            in_data = torch.rand(64, 4).cuda()
-            in_data.requires_grad = True
-            if input_cls is list:
-                in_data = [in_data]
-            else:
-                assert input_cls is dict
-                in_data = {"in": in_data}
 
-            out = model(in_data)
-            out.sum().backward()
-            optim.step()
-            optim.zero_grad()
+    world_size = 1
+    result = dist_init(0, world_size, temp_files[0], temp_files[1])
```

Comment:
nit: inline world_size? since it is not used anywhere else

Reply:
Thanks for the quick review! done.